core/crdt/base.go

```diff
@@ -28,13 +28,26 @@ import (
 type baseCRDT struct {
 	store datastore.DSReaderWriter
 	key   core.DataStoreKey
+
+	// schemaVersionKey is the schema version datastore key at the time of commit.
+	//
+	// It can be used to identify the collection datastructure state at time of commit.
```

Comment:
typo: "at the time"

Note: I don't care about this level or correctness, but it doesn't match the sentence above :)

Reply:
✅